demo/info_section.js

```diff
@@ -196,11 +195,11 @@ shakaDemo.updateLanguageOptions_ =
   var selectedTrack = activeTracks[0];
 
   // Populate list with new options.
-  languages.forEach(function(lang) {
+  tracks.forEach(function(track) {
```

Comment:
`updateLanguageOptions_` updates the list of languages in the demo.  You should not be changing this method.

Instead, you should be changing the formatting in the list of tracks.

Everything else looks good, so I'm going to go ahead and run tests on the build bot.  If all tests pass, I will fix the demo UI myself when I merge.